allensdk/brain_observatory/behavior/data_objects/cell_specimens/traces/dff_traces.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+import numpy as np
+from pynwb import NWBFile
+from pynwb.ophys import DfOverF
+
+from allensdk.brain_observatory.behavior.data_files.dff_file import DFFFile
+from allensdk.brain_observatory.behavior.data_objects import DataObject
+from allensdk.brain_observatory.behavior.data_objects.base \
+    .readable_interfaces import \
+    DataFileReadableInterface, NwbReadableInterface
+from allensdk.brain_observatory.behavior.data_objects.base\
+    .writable_interfaces import \
+    NwbWritableInterface
+from allensdk.brain_observatory.behavior.data_objects.timestamps\
+    .ophys_timestamps import \
+    OphysTimestamps
+
+
+class DFF_traces(DataObject, DataFileReadableInterface, NwbReadableInterface,
```

Comment:
Is there a reason this is the only class named with snake case instead of pascal?